### ISSUE TITLE
Model overfitting and not learning

### ISSUE THREAD

**Issue by sadrasafa** [tom-roddick/mono-semantic-maps#20]
I used your script and generated the ground truth labels and then used your code to train the model. However, the model is only overfitting. Train IoUs increase consistently but validation IoUs do not increase. 

![Screenshot from 2022-09-22 16-06-03](https://user-images.githubusercontent.com/12169967/191754998-d95339aa-57b7-4f64-8f44-65be118cc805.png)
![Screenshot from 2022-09-22 16-06-21](https://user-images.githubusercontent.com/12169967/191755050-65cceb9a-def2-4485-8350-3d1c0a11b03f.png)

**Comment by Junyu-Z** [comment]
Hi, what's your final result? Can you reproduce the **19.1% Mean IOU** reported in their paper?
